ui/self/ui_controls.scs

```diff
@@ -87,8 +87,11 @@ ui_control_search_idtf_edit
 		... (* -> [control that allows to search sc-elements by their identifiers] (* <- lang_en;; *);; *);;
 	*);;
 
-ui_control_search_idtf_edit_example = "file://contents/ui_control_search_idtf_edit_example.gif";;
-
+image
+-> "file://contents/ui_control_search_idtf_edit_example.gif"
+	(*
+		=> nrel_system_identifier: [ui_control_search_idtf_edit_example];;
+	*);;
```

Comment:
```suggestion
"file://contents/ui_control_search_idtf_edit_example.gif"
  <- image;
  <= nrel_system_identifier: [ui_control_search_idtf_edit_example];;
```

Reply:
Yes, the better idea it to start description form an entity being described, not the some another concept